README.md

```diff
@@ -1,25 +1,55 @@
 # DOMException
 An implementation of the DOMException class from NodeJS
 
-NodeJS has DOMException built in (from v10.5), but it is not easily available... you can't require or import it from somewhere (unless you use node v17 - at which point it got added to global state)
+NodeJS has DOMException built in (from v10.5), but it is not easily available... you can't require or import it from somewhere (unless you use node v17 - at which point it got added to global scope)
 
 This package exposes the [`DOMException`](https://developer.mozilla.org/en-US/docs/Web/API/DOMException) class that comes from NodeJS itself. (including all of the legacy codes)
 
+## Install
+
+```bash
+# esm-only (makes use of atob | require NodeJS 16+)
+npm install node-domexception
+
+# cjs (makes use of worker_thread | require NodeJS 10+)
+npm install node-domexception@1.x
+```
+
+v2.x now depend on global `atob` to obtain `DOMException` from a error.
```

Comment:
ah wasnt aware atob domexception error was node 16+ i hoped it was at least node 15